src/engine/controls/cuecontrol.cpp

```diff
@@ -551,13 +553,19 @@ void CueControl::reloadCuesFromTrack() {
     if (!m_pLoadedTrack)
```

Comment:
With most of the logic moved to `trackAnalyzed()`, method `reloadCuesFromTrack()` no longer has any purpose and I suggest removing it. All calls to this method can be replaced with calls to `loadCuesFromTrack()`.

Reply:
Done.